README.md

```diff
@@ -27,6 +27,14 @@ vulnerability details and generating security advisories.
 * sqlite
 * expac
 
+Python dependencies can be installed in a virtual environment (`virtualenv`), by running:
+
+```python
+virtualenv venv
+. venv/bin/activate
+pip install -r requirements.txt
+```
```

Comment:
actually this is not a python code block, this should just use the regular un-syntaxed code block